### ISSUE TITLE
Add K-neighbors classifier and regressor.

### ISSUE THREAD

**Issue by param087** [param087/swiftML#10]
### **K-Neighbors -** 
***
#### `KNeighborsClassifier(neighborCount: 5, weights: "distance", p: 2)`
   - **neighborCount**:  Number of neighbors to use by queries.
   - **weights**: Weight function used in prediction.
   - **p**: Power parameter for the Minkowski metric.

#### API Methods

  **`fit(data: Tensor, labels: Tensor)`**: Fit a K-neighbors classifier model.
  **`prediction(for: Tensor)`**: Predict the class labels for the provided data.
  **`score(data: Tensor, labels: Tensor)`**: Returns the mean accuracy.
***

 ###  **References** 
  - https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm
  - https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html

*** 
#### `KNeighborsRegressor(neighborCount: 5, weights: "distance", p: 2)`
   - **neighborCount**:  Number of neighbors to use by queries.
   - **weights**: Weight function used in prediction.
   - **p**: Power parameter for the Minkowski metric.

#### API Methods

  **`fit(data: Tensor, labels: Tensor)`**: Fit a K-neighbors regressor model.
  **`prediction(for: Tensor)`**: Predict the target value for the provided data.
  **`score(data: Tensor, labels: Tensor)`**: Returns the coefficient of determination $R^2$ for the prediction.

 ###  **References** 
  - https://en.wikipedia.org/wiki/K-nearest_neighbors_algorith
  - https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm#k-NN_regression
  - https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsRegressor.html
